docs/ref/models/expressions.txt

```diff
@@ -255,6 +255,15 @@ is null) after companies that have been contacted::
     from django.db.models import F
     Company.objects.order_by(F('last_contacted').desc(nulls_last=True))
 
+.. versionchanged:: 4.2
+
+    Support for inverting expression ``~F()`` was added.
```

Comment:
I'd mention that it's about conditional expression, or more explicitly _"expressions that output BooleanField"_.

Reply:
I'm not sure if _"inverting"_ is the best wording :thinking:, maybe _"logic negation (``NOT`` operator)"_.